tests/baselines/reference/typedefCrossModule2.symbols

```diff
@@ -14,7 +14,9 @@ var bb;
 
 var bbb = new mod.Baz();
 >bbb : Symbol(bbb, Decl(use.js, 5, 3))
+>mod.Baz : Symbol(Baz)
 >mod : Symbol(mod, Decl(use.js, 0, 3))
+>Baz : Symbol(Baz)

```

Comment:
`mod` is a union; one constituent has `Baz: any` and the other has a string index signature to any (as is commonly provided in JS). The union therefore now has a property `Baz`, although its type is still any.